### PR TITLE
Fix compilation warnings

### PR DESCRIPTION
**Pull request by pennam**
Just a round on compilation warnings

### CODE REVIEW

File: src/utility/time/NTPUtils.h [hunk 1]
```diff
@@ -54,7 +54,7 @@ class NTPUtils
   static int           const MAX_NTP_PORT         = 65535;
 #endif
   static unsigned long const NTP_TIMEOUT_MS       = 1000;
-  static char constexpr *    NTP_TIME_SERVER      = "time.arduino.cc";
+  static constexpr const char * NTP_TIME_SERVER   = "time.arduino.cc";
```
Comment (aentinger) [inline src/utility/time/NTPUtils.h:57]: I wonder if `constexpr const char` is not a bit redundant.

Should not `static constexpr char * NTP_TIME_SERVER = "time.arduino.cc";`be enough?

**Review by aentinger — APPROVED** [review]
LGTM :+1:

File: src/utility/time/NTPUtils.h (hunk 1 above)
Comment (pennam) [reply src/utility/time/NTPUtils.h:57]: I was also thinking so, but after a search about the warning i've foud out that:

`constexpr const char * p ` declares a const pointer to const char data
`constexpr char * p ` declares a const pointer to char data

Comment (aentinger) [reply src/utility/time/NTPUtils.h:57]: Thank you for checking that up ;) Now I also grew a bit smarter =)

**Pull request merged**